app/component/itinerary/ItineraryDetails.js

```diff
@@ -330,6 +340,19 @@ class ItineraryDetails extends React.Component {
                   }
                 />
               )}
+
+              {config.itineraryFollowerOn && (
+                <div>
+                  <SecondaryButton 
+                    ariaLabel="follow" 
+                    buttonName="followtheitinerary"
+                    buttonClickAction={followItinerary}
+                    buttonIcon="icon.icon_mapMarker-via-map"
+                    smallSize /> 
+                </div>
+              )}
```

Comment:
As mentioned. I think it's better to either use <Link> component here directly or probably preferable create a new component which uses the <Link> component. I think you can somewhat mimic where `MobileTicketPurchaseInformation`  is located in this component and how it's done (although it's an external link and the design is slightly different than what the style for this link should be according to https://www.figma.com/design/kt6RO4afqFd8Ej8EmpuWpx .

Reply:
Also, we should probably hide this link in desktop view completely.